pandas/computation/pytables.py

```diff
@@ -129,7 +129,7 @@ def conform(self, rhs):
         """ inplace conform rhs """
         if not com.is_list_like(rhs):
             rhs = [rhs]
-        if hasattr(self.rhs, 'ravel'):
+        if hasattr(rhs, 'ravel'):
```

Comment:
So are you saying it should be `if isinstance(rhs, np.ndarray)`?  That seems to work okay.